spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -354,45 +354,6 @@ def f(x):
         assert content['found']
 
 
-def test_runcell(tmpdir):
```

Comment:
Is it possible to add a new test that replaces this one?

Reply:
Well this test was not removed, it was just transferred to spyder. This makes sense because runcell doesn't work without a spyder frontend.